### ISSUE TITLE
Fritzing compiles/runs in QT but fails Visual Studio 2019 release

### ISSUE THREAD

**Issue by fscii** [fritzing/fritzing-app#3818]
Since it runs in QT, libgit2.dll and everything should be fine.  I've updated the path in the release_fritzing.bat file and as you can see from the screenshot libgit2 is found but it bombs at the linker stage.  
I run the bat file with 3 parameters: 0.9.6b 64 2019 in the Visual Studio cmd shell

Help please.

<please attach screenshot if possible>
![Fritzing_compile error](https://user-images.githubusercontent.com/25595547/116890018-33da0800-abfb-11eb-992e-42db93da91a3.jpg)

Fritzing Version 0.9.6 [Qt 5.15.2] Visual Studio 2019 Windows 10

**Comment by KjellMorgenstern** [comment]
What are you trying to do? Is there any particular issue in Fritzing you want to work on?

**Comment by fscii** [comment]
I am trying to compile a release version in Windows.  It bombs on the final step.

Side note:
I did come across one error in:
https://github.com/fritzing/fritzing-app/wiki/1.-Building-Fritzing

Under windows:
cmake .. -G "Visual Studio 16 2019" -A x64 -DBUILD_SHARED_LIBS=OFF -DBUILD_CLAR=OFF

Shared Libs must = ON or the resulting file is git2.lib rather than git2.dll

I solved that to get it to work in QT, now I just need to see how to fix this problem in building the final release.
